libs/@local/hql/cst/src/snapshots/hql_cst__expr__test__fn_empty.snap

```diff
@@ -0,0 +1,7 @@
+---
+source: libs/@local/hql/cst/src/expr.rs
+expression: result
```

Comment:
Any way to prettify the description of snapshots? If that's too much work we can leave it.

Reply:
changed in [95dea81](https://github.com/hashintel/hash/pull/4753/commits/95dea817262ed76ed132705fb0cffafe2ec456a7)

Reply:
Much better, thanks!